### ISSUE TITLE
Specify selected payment method `STPPaymentOptionsViewController`

### ISSUE THREAD

**Issue by Drusy** [stripe/stripe-ios#1285]
## Summary
It's impossible to specify `STPPaymentOptionsViewController` selected payment method using `presentPaymentOptionsViewController`.
As the default source|method is not managed anymore by Stripe server, it would be nice to be able to specify the selected payment method in case we would like to create this "default payment" or "last payment method" feature locally.

This feature is available on the Android SDK: https://stripe.dev/stripe-android/com/stripe/android/PaymentSession.html#presentPaymentMethodSelection-boolean-java.lang.String-
```
public void presentPaymentMethodSelection(boolean shouldRequirePostalCode,
                                          @Nullable
                                          java.lang.String userSelectedPaymentMethodId)
```

## Code to reproduce
`paymentContext.presentPaymentOptionsViewController()`

## SDK version
Carthage `github "stripe/stripe-ios" "v16.0.1"`

**Comment by yuki-stripe** [comment]
Hi @Drusy 

Check out https://stripe.dev/stripe-ios/docs/Classes/STPPaymentContext.html#/c:objc(cs)STPPaymentContext(py)defaultPaymentMethod - and be sure to follow the note too (`Set this property immediately after initializing STPPaymentContext, or call retryLoading afterwards.`)

**Comment by Drusy** [comment]
Hi @yuki-stripe

This feature has been released recently in 16.0.2 and I didn't notice sorry!
Going to try this tomorrow and give you a feedback.

**Comment by Drusy** [comment]
Works like a charm, I should have seen this before, thanks again.
